rclcpp/test/rclcpp/test_qos_event.cpp

```diff
@@ -217,6 +237,11 @@ TEST_F(TestQosEvent, test_default_incompatible_qos_callbacks)
 }
 
 TEST_F(TestQosEvent, construct_destruct_rcl_error) {
+  std::string rmw_implementation_str = std::string(rmw_get_implementation_identifier());
```

Comment:
If we update this test along with `execute`, `add_to_wait_set `, `test_on_new_event_callback` tests below to set the event_type to a matched event instead of `RCL_PUBLISHER_OFFERED_DEADLINE_MISSED`, we should be able to include these test for `rmw_zenoh` since they only checks for construction/destruction.

Reply:
@ahcorde what do you think?